src/HealthChecks.Qdrant/QdrantHealthCheck.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Qdrant.Client;
+
+namespace HealthChecks.Qdrant;
+
+/// <summary>
+/// A health check for Qdrant services.
+/// </summary>
+public class QdrantHealthCheck : IHealthCheck
+{
+    private readonly QdrantClient _client;
+
+    public QdrantHealthCheck(QdrantClient? client)
```

Comment:
```suggestion
    public QdrantHealthCheck(QdrantClient client)
```

This shouldn't be nullable if we are going to throw when it is null.